e2etest/bgptests/bgp.go

```diff
@@ -1005,6 +1005,19 @@ var _ = ginkgo.Describe("BGP", func() {
 
 				resources := metallbconfig.ClusterResources{
 					Peers: metallb.PeersForContainers(FRRContainers, ipFamily),
+					Communities: []metallbv1beta1.Community{
```

Comment:
By doing this, we are not testing metallb anymore with communities written in the advertisement.
I would change the table to accept:
- the adv community content (that can either be the alias or the community itself)
- the (optional) []metallbv1beta1.Community to add to resources

In this way, you'll be able to add a couple of entries that test the community crd AND the previous implementation (I'd avoid duplicating all the combinations).

Reply:
Sure, added for ipv4, ipv6, and ipv4 legacy.